src/routes/users/delete.ts

```diff
@@ -0,0 +1,28 @@
+import { APIRoute } from '../../types/API';
+import { HTTPMethod } from '../../types/HTTP/http.status';
+import { Request, Response, NextFunction } from 'express';
+
+import { APIError } from '../../lib/utils/api-error';
+import { ApiErrorCode } from '../../types/HTTP/http.model';
+import { deleteUser } from '../../services';
+import { StatusCodes } from 'http-status-codes';
+
+export default {
+  method: HTTPMethod.DELETE,
+  url: '/user/:id',
+  controller: async (req: Request, res: Response, next: NextFunction) => {
+    const { id } = req.params;
```

Comment:
mozesz dodac sprawdzenie czy id jest uuid a jak nie to wypierdalac odrazu blad zeby bazy bez celu nie przeszukiwac